### PR TITLE
Update monitoring docs and make them slightly more practical.

### PR DESCRIPTION
**Pull request by kzemek**

### CODE REVIEW

**Comment by fenek** [comment]
:+1: Waiting for @goddammit review.

File: doc/operation-and-maintenance/Logging-&-monitoring.md [hunk 1]
```diff
@@ -84,60 +84,47 @@ By default, the resolution is set to 60 seconds.
...
-    $ docker inspect graphite | grep IPAdd | grep -v Secon | cut -d '"' -f 4 | head -n 1
-    172.17.0.2
+Go to http://localhost to view Grafana dashboard that's already set up to use metrics from Graphite.
 
-Get IP address of the machine:
+### Add metrics to Grafana dashboard
 
-    $ docker-machine ip
-    192.168.99.100
+We recommend a baseline of following metrics to track in your MongooseIM instance.
```
Comment (goddammit) [inline doc/operation-and-maintenance/Logging-&-monitoring.md]: alt:
We recommend the following metrics as a baseline for tracking your MongooseIM instance.

File: doc/operation-and-maintenance/Logging-&-monitoring.md [hunk 2]
```diff
@@ -84,60 +84,47 @@ By default, the resolution is set to 60 seconds.
...
-### Run graphite in Docker - alternative method
+![An example graph in Grafana](example-grafana-graph.png)
 
-If one of these steps doesn't work for you (e.g. they may be incompatible with your Docker on Mac), please try a less advanced alternative.
-It doesn't require a docker machine and will expose Graphite ports on localhost, instead of using a new route.
+This screenshot shows a graph plotting the RDBMS simple query time metric mentioned above.
+The graph is plotted for three nodes with each node having a different prefix: `mongoose.node1`, `mongoose.node2` and `mongoose.node3`.
 
-Start a container with Graphite:
+The queries take metrics for all nodes and all domains (`**` is a wildcard for multiple parts of the metric name) and groups them *per-node*, *per-value-type* - respectively `1`st (indexed from `0`) and `-1`st (last) parts of the metric's name.
```
Comment (goddammit) [inline doc/operation-and-maintenance/Logging-&-monitoring.md]: The queries take metrics for all nodes and all domains (`**` is a wildcard for multiple parts of the metric name) and group them *per-node* and *per-value-type* (respectively `1`st and `-1`st part of the metric's name). The parts of the names are indexed from 0.

File: doc/operation-and-maintenance/Logging-&-monitoring.md [hunk 3]
```diff
@@ -84,60 +84,47 @@ By default, the resolution is set to 60 seconds.
...
 
-    $ docker run -d --name graphite --restart=always -p 80:80 -p 2003-2004:2003-2004 -p 2023-2024:2023-2024 -p 8125:8125/udp -p 8126:8126 hopsoft/graphite-statsd
+Because time-based metrics in MongooseIM are given in **microseconds**, to display understandable values in graph's legend the Y-axis unit has to be changed on the `Axes` tab.
```
Comment (goddammit) [inline doc/operation-and-maintenance/Logging-&-monitoring.md]: alt:
Time-based metrics in MongooseIM are given in **microseconds**, so to display human-readable values in graph's legend, the Y-axis unit has to be edited in the`Axes`tab.

**Pull request merged**